lang/cs/Org.Apache.REEF.Client/YARN/IApplicationReport.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This and its implementation seem superfluous to me. Its just subsetting the already existing Application class.